themes/digital.gov/layouts/partials/core/get_related.html

```diff
@@ -80,73 +80,6 @@ <h4 class="dg-promos__heading">Join a Community</h4>
 {{ end }}
 {{ $services := ($.Scratch.Get "services") }}
 
-{{/* If there are services... */}}
-{{- with $services -}}
-  <section id="related_services" class="dg-promos dg-related-services">
-    <h4 class="dg-promos__heading">Get Started Building</h4>
-
-    <ul class="dg-promos__list">
-      {{- range $services -}}
-
-        {{/* If an external link */}}
-        {{- if or .Params.source_url .Params.source -}}
-
-          {{/* If there is a defined source for this page */}}
-          {{- if .Params.source -}}
-            {{/* Get the source data */}}
-            {{- $source := .Site.GetPage (print "source_" .Params.source ) -}}
-            {{/* If there is a logo defined in the source */}}
-            {{- if $source.Params.logo -}}
-              {{/* get the path for the icon — e.g. 18f-icon.png */}}
-              {{- $src := (printf "/logos/%s%s" $source.Params.logo "-icon.png") -}}
-              {{/* Pass these vars into the list item template (botton of page) */}}
-
-              {{- template "promo" dict "item" . "href" (print .Params.source_url "?dg") "src" $src -}}
-            {{- end -}}
-            {{/* No source? let's attempt to get the favicon for the URL */}}
-          {{- else -}}
-            {{/* get the Favicon via Google Favicon service */}}
-            {{- $src := (print "https://www.google.com/s2/favicons?domain=" .Params.source_url) -}}
-            {{/* Pass these vars into the list item template (botton of page) */}}
-            {{- template "promo" dict "item" . "href" (print .Params.source_url "?dg") "src" $src -}}
-          {{- end -}}
-
-          {{/* If an internal link... */}}
-        {{- else -}}
-
-          {{/* Use the digital.gov logo */}}
-          {{/* Pass these vars into the list item template (botton of page) */}}
-          {{- template "promo" dict "item" . "href" (print .Params.source_url "?dg") "src" (print "/img/digit-50.png") -}}
-
-        {{- end -}}
-
-      {{- end -}}
-
-    </ul>
-
-    <footer>
-      <p>
-        <a href="{{ "/services/" | absURL }}" class="dg-more-link"
-          ><span
-            >See all
-            {{ len (where $.Site.RegularPages "Section" "services") }}
-            services</span
-          >
-          <svg
-            class="usa-icon dg-icon dg-icon--small margin-bottom-05"
-            aria-hidden="true"
-            focusable="false"
-          >
-            <use
-              xlink:href="{{ "uswds/img/sprite.svg#arrow_forward" | relURL }}"
-            ></use>
-          </svg>
-        </a>
-      </p>
-    </footer>
-  </section>
-{{- end -}}
-
 {{- define "promo" -}}
```

Comment:
Can you confirm if lines 83-120 is still needed?

Reply:
They are not needed.. after removing I didnt see any regression nor could i find it in the dom when inspecting as well